monai/deploy/graphs/__init__.py

```
@@ -1 +1,11 @@
-from holoscan.graphs import *
+"""Graph symbols exported from Holoscan when available.
+
+Some Holoscan package variants/environments do not ship ``holoscan.graphs``.
+Importing this module should remain non-fatal so ``import monai.deploy`` works
+for tooling/style checks where graph APIs are not used.
+"""
+
+try:
+    from holoscan.graphs import *  # noqa: F401,F403
+except Exception:  # pragma: no cover - environment dependent optional import
```

Ruff (0.15.12)

[warning] Do not catch blind exception: `Exception`

(BLE001)